### PR TITLE
Update footer.html

### PR DESCRIPTION
**Pull request by lizygeogy**

### CODE REVIEW

**Comment by lizygeogy** [comment]
@wvary 
I have updated the footer with the required info. Please check and merge.

**Comment by wvary** [comment]
@lizygeogy, Your change does not look anything like the our website.

Our website footer:
image

Change in this PR:
image

**Comment by mohini-sri90** [comment]
Updated code results in the following - 
footer-docs

**Comment by mohini-sri90** [comment]
@wvary Please have a look and let me know if the styling looks okay

File: docs/custom/readthedocs_dt/footer.html [hunk 1]
```diff
@@ -21,5 +21,6 @@
     {% endif %}
   </div>
 
-  Built with <a href="http://www.mkdocs.org">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
+  <div class="footer">Copyright © 2018 DataTorrent</div>
```
Comment (wvary) [inline docs/custom/readthedocs_dt/footer.html]: @lizygeogy Please change copy right symbol to &#169;

**Comment by wvary** [comment]
@lizygeogy The new text looks right, but I feel the font color is a bit prominent for a footer.  How about if we keep the color as #999 as before.  I like that you changed the font-size a bit smaller.

**Comment by mohini-sri90** [comment]
@wvary Please review and merge when you see fit

**Pull request merged**